SU-SWS/stanford_starter
# ADOPTD8-38: Added common subtheme structure for easy customization.

# READY FOR REVIEW

# Summary
- Add in empty files and structure for easy subtheming

# Review By (Date)
- Whenever convenient, no rush

# Urgency
- low 

# Steps to Test

1. Pull down branch and see added utilities folder
2. see files in scss/theme folder as example where to style buttons etc

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- ADOPTD8-38

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
